sds_data_manager/stacks/ialirt_database_stack.py

```diff
@@ -0,0 +1,52 @@
+"""Stack for creating the IALIRT database."""
+
+from aws_cdk import (
+    RemovalPolicy,
+    Stack,
+)
+from aws_cdk import (
+    aws_dynamodb as ddb,
+)
+from constructs import Construct
+
+
+class IAlirtDatabaseStack(Stack):
+    """Stack for creating the IALIRT database."""
+
+    def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
+        """Initialize the IALIRT database stack.
+
+        Parameters
+        ----------
+        scope : Construct
+            The App object in which to create this Stack
+        construct_id : str
+            The ID (name) of the stack
+        **kwargs
+            Additional keyword arguments.
+
+        Note:
+        DynamoDB's table defines the key schema and any secondary indexes.
+        The non-key attributes are included items are added to the table.
+        """
+        super().__init__(scope, construct_id, **kwargs)
+
+        self.packet_data_table = ddb.Table(
+            self,
+            "PacketDataTable",
+            table_name="imap-packetdata-table",
+            # Change to RemovalPolicy.RETAIN to keep the table after stack deletion.
+            removal_policy=RemovalPolicy.DESTROY,
+            # Restore data to any point in time within the last 35 days.
+            point_in_time_recovery=False,
+            # Partition key (PK) = spacecraft time ugps.
+            partition_key=ddb.Attribute(
+                name="sct_vtcw_reset#sct_vtcw",
+                type=ddb.AttributeType.STRING,
+            ),
```

Comment:
Do we need to stick to ops-software shorthand names or can we be more descriptive in our names (what is ugps? We don't have GPS on IMAP, so is it equivalent to an earth received time, or MET, something else...). 
Also, do we want this as the PK alone, or do you want to split this into `sct_vtcw_reset` as the PK, and `sct_vtcw` as the SK?

```suggestion
            # Partition key (PK) = spacecraft time ugps.
            partition_key=ddb.Attribute(
                name="spacecraft_time",  # MET?
                type=ddb.AttributeType.STRING,
            ),
```

Reply:
Ah, that's true. It would be MET. I can make that change. Let's discuss database layout in Slack. I could see us going back and forth for that.

Reply:
We want to have High Cardinality: break up the partition key in as small pieces as possible.